pkg/controller/cassandradatacenter/controller/statefulset.go

```diff
@@ -0,0 +1,185 @@
+package controller
+
+import (
+	"context"
+	cassandraoperatorv1alpha1 "github.com/instaclustr/cassandra-operator/pkg/apis/cassandraoperator/v1alpha1"
+	"k8s.io/api/apps/v1beta2"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func CreateOrUpdateStatefulSet(reconciler *CassandraDataCenterReconciler, cdc *cassandraoperatorv1alpha1.CassandraDataCenter, volumeMounts VolumeMounts) (*v1beta2.StatefulSet, error) {
+
+	dataVolumeClaim := getDataVolumeClaim(cdc.Spec.DataVolumeClaimSpec)
```

Comment:
I don't particularly like the name `get` as it implies fetching the object from somewhere, instead of what it's actually doing which is instantiating an object. Perhaps `new` would be a better name?

Reply:
prefixed with `new` instead of `get`